app/models/step_by_step_page.rb

```diff
@@ -1,4 +1,6 @@
 class StepByStepPage < ApplicationRecord
+  has_many :steps, dependent: :destroy
```

Comment:
We might want to use something like `has_many :steps, -> { order(position: :asc) }` to ensure they come back in the right order.  We'd want to add a test for that too, so I'm happy for this to happen soon

Reply:
Thanks @sihugh , I've made a note to remember to add this after 👍